services/identity/service.go

```diff
@@ -110,7 +110,7 @@ func (s *Service) CreateIdentity(si *network.ServerIdentity, ai *CreateIdentity)
 
 	roster := ids.Root.Roster
 	replies, err := manage.PropagateStartAndWait(s.Context, roster,
-		&PropagateIdentity{ids}, 1000, s.Propagate)
+		&PropagateIdentity{ids}, 10000, s.Propagate)
```

Comment:
It's very specific for that service and shouldn't change during runtime, so I'll put in a constant.